pkg/eval/math/math.go

```diff
@@ -76,6 +76,22 @@ import (
 // ▶ (float64 -1)
 // ```
 
+//elvdoc:fn cosh
+//
+// ```elvish
+// math:cosh $number
+// ```
+//
+// Computes the hyperbolic cosine of `$number` in units of radians (not degrees).
+// Examples:
+//
+// ```elvish-transcript
+// ~> math:cosh 0
+// ▶ (float64 1)
+// ~> math:cosh $math:pi
```

Comment:
This is not a particularly good example since pi is not special in hyperbolic trigonometric functions.

Reply:
Good point, that was just copy/paste without much thought about what would constitute good examples for these functions. I'll follow the Go math documentation here.